internal/http_client.go

```diff
@@ -441,3 +443,9 @@ func retryNetworkAndHTTPErrors(statusCodes ...int) RetryCondition {
 		return false
 	}
 }
+
+// GetMetricsHeader constructs header value for metrics attribution
+func GetMetricsHeader(sdkVersion string) string {
```

Comment:
Is it possible to access the `Version` in here? That might be better than passing it in from all the APIs separately.

Reply:
Not possible, that would cause a dependency cycle. I'm not a fan of how this is structured either. I would prefer if we kept the Version value separately from firebase.go.